examples/redis_consumer.py

```
@@ -0,0 +1,265 @@
+
+def main():
+    """Main entry point"""
+    # Setup signal handlers
+    signal.signal(signal.SIGINT, signal_handler)
+    signal.signal(signal.SIGTERM, signal_handler)
+    
+    # Parse command line arguments (basic example)
+    import argparse
+    parser = argparse.ArgumentParser(description='Device Event Consumer Example')
+    parser.add_argument('--redis-host', default='localhost', 
+                       help='Redis server hostname (default: localhost)')
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)